3_javascript/javascript1/week1/Goodboy-Oldboy/script.js

```diff
@@ -0,0 +1,15 @@
+
+// Info of Year
+const dogYearOfBirth = 2023;
+const dogYearFuture= 2060;
+
+//calculation
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+// dog or not dog
+const shouldShowResultInDogYears = true ;
+
+// Answer
+if(shouldShowResultInDogYears){ console.log("Your dog will be " + (dogYear * 7) + " dog years old in " + dogYearFuture)}
```

Comment:
try to use template literals for console.log and code alignments: 
if (shouldShowResultInDogYears) { 
    console.log("Your dog will be " + (dogYear * 7) + " dog years old in " + dogYearFuture)}
}